package.json

```diff
@@ -29,13 +29,11 @@
         "analyze": "cross-env ANALYZE=true next build",
         "build": "cross-env NODE_ENV=production next build",
```

Comment:
On my site, I removed `cross-env` because I dropped `NODE_ENV=production` from build & start. I also removed the bundle analyzer on mine to simplify `next.config.js`.  

I've also been debating removing the `imageoptim` process from the repo directly in favor of [Imgbot](https://github.com/marketplace/imgbot/plan/MDIyOk1hcmtldHBsYWNlTGlzdGluZ1BsYW4xNzQ5) but I'm not sure yet.

Reply:
> On my site, I removed cross-env because I dropped NODE_ENV=production from build & start. I also removed the bundle analyzer on mine to simplify next.config.js.

I will likely do the same.

> I've also been debating removing the imageoptim process from the repo directly in favor of Imgbot but I'm not sure yet.

I swear I have an open tab somewhere with an idea for this, but I cannot find it for the life of me now... It was either a GitHub action or a ZEIT integration that replaced the whole thing. Here's to hoping I find it eventually. 🤞